### ISSUE TITLE
Add mempool tools 

### ISSUE THREAD

**Issue by whoabuddy** [aibtcdev/aibtcdev-backend#6]
We could add a tool that allows the agent to check the mempool for all TXs, or to see it's own TXs.

**Comment by Rapha-btc** [comment]
I know a man for this job @tripnmonkey

**Comment by codeonedotzero** [comment]
Website started... right now it is all real-time data, but the roadmap over the next few weeks will build out a backend and API.

Maybe sync up on requirements to see how they match up?

Check it out... about a week old and still testing but it is pretty stable if you don't bounce between pages.

https://nakastats.xyz/

**Comment by whoabuddy** [comment]
@codeonedotzero Yeah would love to learn more! Saw this drop about a week ago and love the idea of exposing mempool costs in a more visual way. Knowing which contract transactions cost a lot is great for developers to look for opportunities to optimize.

As far as agent tooling, we can [cache your API responses](https://github.com/aibtcdev/aibtcdev-cache) and create some scripts that create context for the agents using the data.

**Comment by Rapha-btc** [comment]
It's coming together smoothly.

**Comment by whoabuddy** [comment]
Closing as stale - can revisit later if needed.
